dts/xtensa/esp32.dtsi

```diff
@@ -0,0 +1,43 @@
+#include "skeleton.dtsi"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "LX6";
+			reg = <0>;
+		};
+
+		cpu@1 {
+			device_type = "cpu";
+			compatible = "LX6";
+			reg = <1>;
+		};
+
+	};
+
+	sram0: memory@0xbe000000 {
```

Comment:
remove 0x, also node address and reg don't match

Reply:
done